### ISSUE TITLE
Piped commands don't work on OS X

### ISSUE THREAD

**Issue by dfroger** [heimdali/heimdali#3]
The commands:

```
so $HEIMDALI_DATA_DIR/imtest_z5_y4_x3_c2.h5 $HEIMDALI_DATA_DIR/imtest_z5_y4_x3_c2.h5 | bi -n 1 | sc -n 2 > pipe1_out.h5
h5dump pipe1_out.h5
```

Produce the error:

```
h5dump error: unable to open file "pipe1_out.h5"
```

**Comment by dfroger** [comment]
Heimdali version: 0.5.0

**Comment by dfroger** [comment]
Fix with 523850a16a6a669153884a3bb3af0e1f386ac767
